datadog/resource_datadog_integration_aws.go

```diff
@@ -110,6 +111,7 @@ func resourceDatadogIntegrationAwsPrepareCreateRequest(d *schema.ResourceData, a
 		for _, s := range attr.([]interface{}) {
 			hostTags = append(hostTags, s.(string))
 		}
+		iaws.SetHostTags(hostTags)
```

Comment:
Do you know if this changes the scenario when these fields aren't set in the TF config? I'm wondering if a null value gets sent over an empty list, and if that makes a difference in behavior (since we're now setting this field in the if branch instead of always)